frontend/WikiContrib-Frontend/src/result.js

```diff
@@ -567,130 +559,101 @@ class QueryResult extends React.Component {
-                              let start_time = new Date(
-                                updated_val.getFullYear(),
-                                updated_val.getUTCMonth() - incr,
-                                1
-                              );
-                              let month = start_time.getUTCMonth() + 1;
-                              filters.start_time =
-                                start_time.getFullYear() + '-' + month + '-01';
-                              this.setState({
-                                update_filters: filters,
-                              });
-                            }}
-                            placeholder="Select Date"
-                            closeOnChange={true}
-                          />
-                        </Grid.Column>
-                        <Grid.Column computer={8} tablet={16} mobile={16}>
-                          <Header>Time Range</Header>
-                          <Dropdown
-                            style={{ marginTop: 10 }}
-                            fluid
-                            search
-                            selection
-                            icon={false}
-                            options={filter_2}
-                            value={get_timestamp(
-                              new Date(uf.start_time),
-                              new Date(uf.end_time)
-                            )}
-                            onChange={(e, obj) => {
-                              let date = new Date(
-                                this.state.update_filters.end_time
-                              );
-                              let value = obj.value;
-                              let incr =
-                                value <= 31
-                                  ? 1
-                                  : value <= 61
-                                  ? 2
-                                  : value <= 92
-                                  ? 3
-                                  : value <= 183
-                                  ? 6
-                                  : 12;
-                              date = new Date(
-                                date.getFullYear(),
-                                date.getUTCMonth() - incr,
-                                1
-                              );
-                              let month = date.getUTCMonth() + 1;
-                              let filters = Object.assign({}, uf);
-                              filters.start_time =
-                                date.getFullYear() + '-' + month + '-' + 1;
-                              this.setState({ update_filters: filters });
-                            }}
-                            placeholder="Get by date"
-                            closeOnChange={true}
-                          />
-                        </Grid.Column>
-                      </Grid.Row>
+                            let updated_val = new Date(filters.end_time);
```

Comment:
I don't see any new change in the lines 598-656 in comparison with 633-693. No big deal, but in case you want to re-examine.

Reply:
I reformatted that code block after removing a closing tag  </Grid.Row> that belonged to the JSX I removed above. Maybe there was a better way of doing it though.